src/containers/collectibles/components/CollectiblesPage.tsx

```diff
@@ -0,0 +1,629 @@
+        )
+      })
+    }
+    // eslint-disable-next-line
+  }, [])
+
+  const handleEditClick = useCallback(() => {
+    if (isMobile) {
+      setShowUseDesktopDrawer(true)
+    } else {
+      setIsEditingPreferences(true)
+    }
+  }, [isMobile, setShowUseDesktopDrawer, setIsEditingPreferences])
+
+  const handleDoneClick = useCallback(() => {
+    setIsEditingPreferences(false)
+    if (updateProfile) {
+      updateProfile({
+        ...profile,
+        has_collectibles: true,
+        collectibles: { ...collectiblesMetadata }
+      })
+    }
+  }, [setIsEditingPreferences, updateProfile, profile, collectiblesMetadata])
+
+  const handleShowCollectible = useCallback(
+    (id: string) => () => {
+      setCollectiblesMetadata({
+        ...collectiblesMetadata,
+        order: (collectiblesMetadata?.order ?? []).concat(id)
+      })
+    },
+    [setCollectiblesMetadata, collectiblesMetadata]
+  )
+
+  const handleHideCollectible = useCallback(
+    (id: string) => () => {
+      setCollectiblesMetadata({
+        ...collectiblesMetadata,
+        [id]: collectiblesMetadata?.id ?? {},
+        order: (collectiblesMetadata?.order ?? []).filter(
+          tokenId => tokenId !== id
+        )
+      })
+    },
+    [setCollectiblesMetadata, collectiblesMetadata]
+  )
+
+  const onDragEnd = (result: any) => {
+    const { source, destination } = result
+
+    if (!destination || destination.index === source.index) {
+      return
+    }
+
+    let newCollectibleList = getVisibleCollectibles()
+    const sourceCollectible = newCollectibleList.splice(source.index, 1)[0]
+    newCollectibleList = newCollectibleList
+      .slice(0, destination.index)
+      .concat(sourceCollectible)
+      .concat(newCollectibleList.slice(destination.index))
+
+    setCollectiblesMetadata({
+      ...collectiblesMetadata,
+      order: newCollectibleList
+        .map(c => c.id)
+        .filter(id => (collectiblesMetadata?.order || []).includes(id))
+    })
+  }
+
+  const getVisibleCollectibles = useCallback(() => {
+    if (collectiblesMetadata?.order === undefined) {
+      return [...collectibleList]
+    }
+
+    const collectibleMap: {
+      [key: string]: Collectible
+    } = collectibleList.reduce((acc, curr) => ({ ...acc, [curr.id]: curr }), {})
+    const collectibleKeySet = new Set(Object.keys(collectibleMap))
+
+    return collectiblesMetadata.order
+      .filter(id => collectibleKeySet.has(id))
+      .map(id => collectibleMap[id])
+  }, [collectiblesMetadata, collectibleList])
+
+  const getHiddenCollectibles = useCallback(() => {
+    const visibleCollectibleKeySet = new Set(
+      getVisibleCollectibles().map(c => c.id)
+    )
+    return collectibleList.filter(c => !visibleCollectibleKeySet.has(c.id))
+  }, [getVisibleCollectibles, collectibleList])
+
+  return (
+    <div className={styles.collectiblesWrapper}>
+      <div className={styles.wrapper}>
+        <div className={cn(styles.header, { [styles.mobileHeader]: isMobile })}>
```

Comment:
@piazzatron made a helper for mobile styles that could be used. 
My personal preference would be a top level `.mobile` class next to `collectiblesWrapper` and then for these you won't need to add isMobile every time, but instead use a more specified selector ie. `.mobile .header`